docs/references/express/overview.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+
+### Create a Clerk application
+
+You need to create a Clerk application in your Clerk Dashboard before you can set up Clerk Express.js. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+
+### Install `@clerk/express`
+
+To get started using Clerk with Express.js, add the [SDK](https://www.npmjs.com/package/@clerk/express) to your project:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+
+### Set environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+
+</SignedIn>
+
+<SignedOut>
+
+1. Navigate to the Clerk Dashboard.
+2. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+3. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+4. Paste your keys into your `.env.local` file.
+
+The final result should resemble the following:
+
+</SignedOut>
+
+  ```sh filename=".env.local"
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+</Steps>
+
+## Available methods
+
+The Clerk Express SDK exports the following:
+
+- `@clerk/backend` - You can use [Clerk's JavaScript Backend SDK](/docs/references/backend/overview) to interact with the Clerk API.
+- `clerkMiddleware()` - The [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware) helper integrates Clerk authentication into your Next.js application through Middleware.
+- `getAuth()` - The [`getAuth()`](/docs/references/nextjs/get-auth) helper retrieves the authentication state, allowing you to protect your API routes or gather relevant data.
```

Comment:
More of a note for @alexisintech here - this is one of those spots where we bounce you around between SDKs because we don't have good export mapping. Hopefully we can get this fixed down the line, I don't see a lot we can do about it at the moment though.

Reply:
I'll make these look like external links so that the user isn't directed from one SDK to another, but instead, from one tab to another